src/PowerShellEditorServices.Protocol/Server/DebugAdapter.cs

```diff
@@ -118,6 +118,17 @@ protected Task LaunchScript(RequestContext<object> requestContext)
 
         private async Task OnExecutionCompleted(Task executeTask)
         {
+            try
+            {
+                await executeTask;
+            }
+            catch (Exception e)
+            {
+                Logger.Write(
+                    LogLevel.Error,
+                    "Exception occurred while awaiting debug launch task.\n\n" + e.ToString());
```

Comment:
I suggest using string interpolation and use platform independent newlines:
`$"Exception occurred while awaiting debug launch task.{Environment.Newline}{Environment.Newline}{e.ToString()}"`

Reply:
For consistency I'd like to keep this the same for now.  Ultimately we shouldn't be making our own strings while logging at all, that would better be handled by Serilog.

Reply:
That's fine, I just wanted to bring it up as some people are not aware of some of the new C# features

Reply:
You should be able to use `Logger.WriteException` here and it will handle the newlines/indentation.